src/common/modals/JavaSetup.js

```diff
@@ -203,26 +229,40 @@ const AutomaticSetup = () => {
   const [downloadPercentage, setDownloadPercentage] = useState(null);
   const [currentStep, setCurrentStep] = useState('Downloading Java');
   const javaManifest = useSelector(state => state.app.javaManifest);
+  const java16Manifest = useSelector(state => state.app.java16Manifest);
   const userData = useSelector(state => state.userData);
   const tempFolder = useSelector(_getTempPath);
   const dispatch = useDispatch();
 
   const installJava = async () => {
     const javaOs = convertOSToJavaFormat(process.platform);
     const javaMeta = javaManifest.find(v => v.os === javaOs);
+    const java16Meta = java16Manifest.find(v => v.os === javaOs);
     const {
       version_data: { openjdk_version: version },
       binary_link: url,
       release_name: releaseName
     } = javaMeta;
+
+    const {
+      version_data: { openjdk_version: version16 },
+      binary_link: url16,
+      release_name: releaseName16
+    } = java16Meta;
+
     const javaBaseFolder = path.join(userData, 'java');
     await fse.remove(javaBaseFolder);
     const downloadLocation = path.join(tempFolder, path.basename(url));
+    const downloadjava16Location = path.join(tempFolder, path.basename(url16));
 
     await downloadFile(downloadLocation, url, p => {
       ipcRenderer.invoke('update-progress-bar', parseInt(p, 10) / 100);
       setDownloadPercentage(parseInt(p, 10));
     });
+    await downloadFile(downloadjava16Location, url16, p => {
+      ipcRenderer.invoke('update-progress-bar', parseInt(p, 10) / 100);
+      setDownloadPercentage(parseInt(p, 10));
+    });
```

Comment:
This needs to be from 50 to 100 while the java 8 goes from 0 to 50, this way it looks like a unique operation

Reply:
I made a pull at Ladvance to implement a overwiew progress bar - on the way I renamed the steps to `Java.. - ....`. I made this Progess bar exactly because of this.

Reply:
Your PR looks good, just make sure to center the content inside the modal which looks a bit off from the screenshots. Not let's wait for @Ladvace to do some magic

Reply:
Perfectly centered, I didnt screenshot it right :P
![one](https://user-images.githubusercontent.com/76821666/126363037-54e8c5d5-f5e3-4387-9353-a220d51c2f3a.png)
![two](https://user-images.githubusercontent.com/76821666/126363047-7a7ef235-903a-4a36-b861-9960dc47fac6.png)

Reply:
Can we also add the percentage number to the first progress bar?